docs/site.github.md

```diff
@@ -11,9 +11,9 @@ Simply, reference any of the below keys as you would any other site configuratio
 For example, to list a project's name, you might write something like `The project is called {{ site.github.project_title }}` or to list an organization's open source repositories, you might use the following:
 
 ```liquid
-{% for repository in site.github.public_repositories %}
-  * [{{ repository.name }}]({{ repository.html_url }})
-{% endfor %}
```

Comment:
```liquid
<!-- {% raw %} -->

{% for repository in site.github.public_repositories %}
  * [{{ repository.name }}]({{ repository.html_url }})
{% endfor %}

<!-- {% endraw %} -->

Reply:
Oooooo now there's a great idea! That's what I was looking for, @DirtyF!